src/ExpressionEvaluator/Core/Source/FunctionResolver/FunctionResolver.cs

```diff
@@ -3,18 +3,17 @@
 using Microsoft.VisualStudio.Debugger;
 using Microsoft.VisualStudio.Debugger.Clr;
 using Microsoft.VisualStudio.Debugger.ComponentInterfaces;
-using Microsoft.VisualStudio.Debugger.Evaluation;
 using Microsoft.VisualStudio.Debugger.FunctionResolution;
+using Microsoft.VisualStudio.Debugger.Symbols;
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Reflection.Metadata;
-using Microsoft.VisualStudio.Debugger.Symbols;
 
 namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
     internal abstract class FunctionResolver :
```

Comment:
`FunctionResolverBase` is the base class that is also used by the test resolver, to allow writing unit tests. `FunctionResolver` is the base class for language-specific resolvers loaded by the debugger. The unit tests cannot instantiate debugger types so testing is against `FunctionResolverBase` only.